### ISSUE TITLE
Crashs at start of client

### ISSUE THREAD

**Issue by TheButterbrotMan** [spoorn/SpoornPacks#5]
https://bytebin.lucko.me/Oc7aaEmdoR

**Comment by gamer50082** [comment]
can you add like what version of fabric and what version of mc you using + the mods you have

**Comment by TheButterbrotMan** [comment]
https://www.curseforge.com/minecraft/modpacks/deathdusk (2.0.10)

Fabric 0.14.8 + 1.18.2

**Comment by spoorn** [comment]
@TheButterbrotMan I tried 2.0.10 and am able to load into a Single Player world just fine.  Did you have other changes locally?

If you check the `latest.log` file, there may be WARN logs indicating there was a conflict with `injectSPResourcePack` (you can Ctrl+F to find those) that may display what this is conflicting with

**Comment by TheButterbrotMan** [comment]
> injectSPResourcePack

Yes, i update the pack with exception of GlowLib. I cant find "injectSPResourcePack".

**Comment by spoorn** [comment]
It's conflicting with https://www.curseforge.com/minecraft/mc-mods/respro

```
[17:01:59] [Render thread/WARN]: @Redirect conflict. Skipping spoornpacks.mixins.json:ResourcePackManagerMixin from mod spoornpacks->@Redirect::injectSPResourcePack([Ljava/lang/Object;)Lcom/google/common/collect/ImmutableSet; with priority 1000, already redirected by respro.mixins.json:ResourceProviderMixin from mod respro->@Redirect::appendCelibProviders([Ljava/lang/Object;)Lcom/google/common/collect/ImmutableSet; with priority 1000
```

**Comment by spoorn** [comment]
Fixed in 4.2.2-1.18.2+ and 4.3.1-1.19+
